include/alpaka/queue/QueueCpuNonBlocking.hpp

```diff
@@ -73,6 +74,18 @@ namespace alpaka
                     //-----------------------------------------------------------------------------
                     ~QueueCpuNonBlockingImpl() = default;
```

Comment:
We should now remove this destructor as it is missing the virtual keyword and it now has a base class.